modules/dop/providers/projectpipeline/project_source.go

```diff
@@ -31,6 +31,16 @@ type ProjectSourceType interface {
 	GetPipelineCreateRequestV2() string
 }
 
+type PipelineType string
+
+const (
+	cicdPipelineType PipelineType = "cicd"
+)
+
+func (p PipelineType) String() string {
+	return string(p)
+}
+
```

Comment:
the `PipelineType` type should not exist in project_source.go

Reply:
done